docs/commands/graph.query.md

```diff
@@ -848,41 +848,41 @@ This section contains information on all supported functions from the Cypher que
 
 ## Aggregating functions
 
-|Function         | Description|
-| --------------- |:-----------|
-|avg()            | Returns the average of a set of numeric values                                               |
-|collect()        | Returns a list containing all elements which evaluated from a given expression               |
-|count()          | Returns the number of values or rows                                                         |
-|max()            | Returns the maximum value in a set of values                                                 |
-|min()            | Returns the minimum value in a set of values                                                 |
-|sum()            | Returns the sum of a set of numeric values                                                   |
-|percentileDisc() | Returns the percentile of the given value over a group, with a percentile from 0.0 to 1.0    |
-|percentileCont() | Returns the percentile of the given value over a group, with a percentile from 0.0 to 1.0    |
-|stDev()          | Returns the standard deviation for the given value over a group over a sample                |
-|stDevP()         | Returns the standard deviation for the given value over a group over an entire population    |
+|Function                             | Description|
+| ----------------------------------- |:-----------|
+|avg(_expr_)                          | Returns the average of a set of numeric values                                                                                      |
+|collect(_expr_)                      | Returns a list containing all non-null elements which evaluated from a given expression                                             |
+|count(_expr_)                        | Returns the number of non-null values or rows                                                                                       |
+|max(_expr_)                          | Returns the maximum value in a set of values. <br> Note: Map < Node < Relationship < List < Path < String < Boolean < Number < null |
```

Comment:
I don't think we need to specify all type orders, it should be enough saying: "taking into account type ordering"